Formula/pre-commit.rb

```diff
@@ -17,15 +17,16 @@ class PreCommit < Formula
 
   depends_on "libyaml"
   depends_on "python@3.9"
+  depends_on "six"
```

Comment:
you might also want to update `pypi_formula_mappings.json` as well.

Reply:
oops thanks